tools/tuf/download-artifacts/download-artifacts.go

```diff
@@ -368,9 +372,10 @@ func osquerydCommand() *cli.Command {
 		},
 		Action: func(c *cli.Context) error {
 			return downloadComponents("generate-osqueryd-targets.yml", gitBranch, map[string]string{
-				"macos":   "osqueryd.app.tar.gz",
-				"linux":   "osqueryd",
-				"windows": "osqueryd.exe",
+				"macos":       "osqueryd.app.tar.gz",
+				"linux":       "osqueryd",
+				"linux-arm64": "osqueryd-arm64",
```

Comment:
Name of this file has to match what's in `releaser.sh` (which I believe it's just `osqueryd`), right?

Reply:
Am happy to merge this as is and fix/test this when releasing.